src/backend/gog/setup.ts

```diff
@@ -48,33 +46,10 @@ async function setup(
     LogPrefix.Gog
   )
 
-  let commandPrefix = ''
-
   const gameSettings = GameConfig.get(appName).config
   if (!isWindows) {
-    const isCrossover = gameSettings.wineVersion.type === 'crossover'
-    const crossoverBottle = gameSettings.wineCrossoverBottle
-    const crossoverEnv =
-      isCrossover && crossoverBottle ? `CX_BOTTLE=${crossoverBottle}` : ''
-    const isProton = gameSettings.wineVersion.type === 'proton'
-    const { defaultSteamPath } = GlobalConfig.get().getSettings()
-    const prefix = isProton
-      ? `STEAM_COMPAT_CLIENT_INSTALL_PATH="${defaultSteamPath}" STEAM_COMPAT_DATA_PATH='${gameSettings.winePrefix
-          .replaceAll("'", '')
-          .replace('~', userHome)}'`
-      : `WINEPREFIX="${gameSettings.winePrefix
-          .replaceAll("'", '')
-          .replace('~', userHome)}"`
-
-    commandPrefix = `${isCrossover ? crossoverEnv : prefix} ${quoteIfNecessary(
-      gameSettings.wineVersion.bin
-    )} ${isProton ? 'runinprefix' : ''}`
-    // Make sure Proton initialized prefix correctly
-    if (isProton) {
-      await execAsync(
-        `${prefix} ${quoteIfNecessary(gameSettings.wineVersion.bin)} run reg /?` // This is a help command for reg, it's enough to initialize a prefix
-      ).catch()
-    }
+    // Make sure prefix is initalized correctly
+    await verifyWinePrefix(gameSettings)
```

Comment:
i wonder if we should also check if the wine is valid or not, because I imagine, for instance, that a person can start installing a GOG windows game with no wine installed, so we should show the same error we show on `launch` because otherwise the setup will fail and the person won't know the reason.
Check this line, we should do something like this:
https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/blob/01e82586d602e6dedf6a34eaf91b4dfe0fc6120f/src/backend/main.ts#L1015